app/models/publication-flow.js

```diff
@@ -47,6 +46,21 @@ export default class PublicationFlow extends Model {
     return `${this.publicationNumber}`;
   }
 
+  get publicationBeforeDateHasExpired() {
+    return this.publishBefore
+      && moment(this.publishBefore).isBefore(moment());
```

Comment:
Not the core of this PR, but can't we use native javascript date comparison here?

Reply:
Yes, indeed. Although for more complex operations/comparisons I find these kinds of libraries serve a purpose, especially from a semantics/readability point of view. We should lay out a vision on date/time handling for the future of the project I think (getting rid of `moment` will probably be part of that though, so I certainly understand the question, and will remove).